coalib/settings/SectionFilling.py

```diff
@@ -43,8 +50,9 @@ def fill_settings(sections, acquire_settings, log_printer):
 
         local_bears[section_name] = section_local_bears
         global_bears[section_name] = section_global_bears
+        aspects[section_name] = section_aspects
 
-    return local_bears, global_bears
+    return local_bears, global_bears, aspects
```

Comment:
This is a breaking change to the public coala API.
A very large one.
I dont recall this being in an earlier patch I reviewed.